src/drag.ts

```diff
@@ -165,6 +168,11 @@ function processDrag(s: State): void {
         const translation = util.posToTranslateAbs(bounds)(cur.origPos, asWhite);
         translation[0] += cur.pos[0] + cur.dec[0];
         translation[1] += cur.pos[1] + cur.dec[1];
+        // bound piece within screen
+        const maxX = document.body.clientWidth + document.body.scrollLeft - cur.size[0] - 8.5;
+        const maxY = document.body.clientHeight + document.body.scrollTop - cur.size[1] - 8.5;
```

Comment:
What is this `8.5` value?

Reply:
I think it's caused by https://github.com/ornicar/chessground/blob/a722f739a9394b992f233907c6912d24d936502a/src/svg.ts#L251

Reply:
But it only makes sense when multiplying a width, as to reach a center.